install.md

```diff
@@ -1,6 +1,27 @@
 # Installation Instructions
 
-Prior to installing Buildah, install the following packages on your linux distro:
+## System Requirements
+
+### Kernel Version Required
+
+Buildah requires that the kernel be at version 4.0 or higher in order to support the OverlayFS filesystem.
```

Comment:
This is wrong, since it eliminates Centos and RHEL systems, which run fine with the OverlayFS.  Should probably point out that on RHEL and CENTOS 7.4 and greater it also works.

Reply:
OK, I've changed to:

### Kernel Version Requirements
To run Buildah on Red Hat Enterprise Linux or CentOS, version 7.4 or higher is required.
On other Linux distributions Buildah requires a kernel version of 4.0 or
higher in order to support the OverlayFS filesystem.  The kernel version can be checked
with the 'uname -a' command.

Holler if that needs tweaking still.